pkg/cmdlets/auth/login.go

```diff
@@ -178,3 +182,27 @@ func promptToken(credentials *mono_models.Credentials, out output.Outputer, prom
 
 	return nil
 }
+
+// AuthenticateWithDevice attempts to authenticate this device with the Platform.
+func AuthenticateWithDevice(out output.Outputer) error {
+	deviceCode, err := model.RequestDeviceAuthorization()
+	if err != nil {
+		return err
```

Comment:
Please wrap your errors: https://github.com/ActiveState/cli/blob/master/docs/errors.md

(goes for all such cases in this PR)

Reply:
Okay.